wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -73,12 +74,15 @@ namespace :wagn do
 
   desc "migrate structure and cards"
   task :migrate =>:environment do
-    ENV['SCHEMA'] ||= "#{Wagn.root}/db/schema.rb"
-    
+    ENV['SCHEMA'] ||= "#{Cardio.gem_root}/db/schema.rb"
+
     stamp = ENV['STAMP_MIGRATIONS']
 
     puts 'migrating structure'
+    
+    Wagn.paths['db/migrate'] = ["#{Cardio.gem_root}/db/migrate"]
     Rake::Task['db:migrate'].invoke
+    #Rake::Task['wagn:migrate:structure'].invoke
```

Comment:
Here is a serious problem. db:migrate uses the wrong path. The new 'wagn:migrate:structure' task has the problem that it doesn't update schema.rb